Orange/widgets/data/owdatasampler.py

```diff
@@ -241,13 +250,15 @@ def updateindices(self):
                       self.data.domain.has_discrete_class)
         if self.sampling_type == self.FixedSize:
             self.indices = sample_random_n(
-                self.data, self.sampleSizeNumber,
+                self.data, size,
                 stratified=stratified, replace=self.replacement,
                 random_state=rnd)
         elif self.sampling_type == self.FixedProportion:
             self.indices = sample_random_p(
-                self.data, self.sampleSizePercentage / 100,
+                self.data, size,
```

Comment:
sample_random_p takes p as percentage, so size should probably be divided by 100